src/components/StatsPanel/StatsPanel.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+import { describe, it, expect } from 'vitest';
+
+import { StatsPanel } from './StatsPanel';
+
+describe('StatsPanel', () => {
+  it('renders the provided stats correctly', () => {
+    render(<StatsPanel totalTests={32} passed={12} failed={19} waiting={1} />);
```

Comment:
The test renders the component with numeric values but there is no check for accessibility attributes (e.g., roles) or labels. The test relies entirely on getByText for values which can be brittle and not representative of accessible usage.

**Sugerencia:**
```typescript
Prefer semantic queries such as getByRole, getByLabelText or add data-testid attributes if necessary. Also assert the presence of labels alongside values. Example change for the test:

// in component: <StatValue aria-label={`stat-value-${stat.id}`}>{stat.value}</StatValue>

// in test:
expect(screen.getByLabelText('stat-value-total')).toHaveTextContent('32');
expect(screen.getByText('Passed')).toBeInTheDocument();

This makes tests more robust and aligned with accessibility.
```